src/Tizen.NUI/src/public/DragAndDrop/DragAndDrop.cs

```diff
@@ -123,7 +125,7 @@ public void StartDragAndDrop(View sourceView, View shadowView, DragData dragData
                 shadowHeight = MinDragWindowHeight;
             }
 
-            mDragWindow = new Window("DragWindow", new Rectangle(-shadowWidth, -shadowHeight, shadowWidth, shadowHeight), true)
+            mDragWindow = new Window("DragWindow", new Rectangle(dragWindowOffsetX, dragWindowOffsetY, shadowWidth, shadowHeight), true)
```

Comment:
Minor question. After this patch, the default offset changed from "-shadowSize" to "0". Mean, default behaviour changed. Is it okay??